.github/workflows/ci.yml

```
@@ -138,9 +142,7 @@ jobs:
 
   nightly-release:
     runs-on: ubuntu-latest
-    # Lawnchair-TODO(Configuration): Don't publish nightly build until Android 8.0, 8.1 is stable, and (probably) stable to match Lawnchair 15 Beta 1 or 2
-    if: false
-    # if: github.repository_owner == 'LawnchairLauncher' && github.event_name == 'push' && github.ref == 'refs/heads/15-dev'
+    if: env.PUBLISH_NIGHTLY == 'true' && github.repository_owner == 'LawnchairLauncher' && github.event_name == 'push' && startsWith(github.ref, 'refs/heads/') && (endsWith(github.ref, '-dev')
```

actionlint (1.7.12)

[error] unexpected end of input while parsing closing ')' of nested expression (...). expecting ")"

(expression)